Employee.sql

```diff
@@ -0,0 +1,66 @@
+CREATE DATABASE TASK1;
+USE TASK1;
+
+CREATE TABLE Employee(
+EmployeeID INT PRIMARY KEY,
+FirstName VARCHAR(50) NOT NULL,
+LastName VARCHAR(50) NOT NULL,
+Department VARCHAR(30),
+Salary DECIMAL(10,2));
+
+INSERT INTO  Employee VALUES
+(1,'Gokul','Das','IT',700000),
+(2,'Akhil','Philip','Finance',500000),
+(3,'Sangeeth','Kumar','HR',600000);
+
+-- QUERY TO PRINT ALL DATA
+SELECT *FROM Employee;
+
+-- QUERY TO SELECT A COLUMN
+SELECT Department FROM Employee;
+
+-- QUERY TO FILTER WITH WHERE RESULT
+SELECT * FROM Employee WHERE Salary>500000;
+
+-- WHAT IS THE PURPOSE OF THE 'IDENTITY' KEYWORD IN THE CREATE TABLE STATEMENT?
+-- IDENTITY Keyword can be used to auto-increment in a column mainly PRIMARY KEY.
+
+-- QUERY TO SELECT FirstName and Salary ONLY
+SELECT Firstname,Salary FROM Employee;
+
+-- HOW WOULD YOU MODIFY THE EXISTING UPDATE STATEMENT TO GIVE ALL EMPLOYEES IN THE IT DEPARTMENT A 10% RAISE
+UPDATE Employee
+SET Salary = Salary * 1.10
```

Comment:
What will be the new salary of the employee ? Is this correct ?

Reply:
Why its 1.10 ?

Reply:
Like if current salary is 100 and hike is 10%.
new salary will be old salary + 10% of old salary.
new salary = 100 + 0.1*100
                   = 100 + 10
                   = 110
similarly 100 * 1.10 = 110

Reply:
nice !!